frontend/src/components/Forms/FormLayout/index.js

```diff
@@ -10,7 +11,7 @@ const FormLayout = ({ children, headerText, subtitleText }) => {
         <Heading>{headerText}</Heading>
       </div>
       <div className={styles.subtitleWrapper}>
-        <Heading level={4}>{subtitleText}</Heading>
+        <Body>{subtitleText}</Body>
       </div>
```

Comment:
Is it still a `SubTitle` then? Or should the `Body` component be a `SubTitle` component?